packaging/hudi-flink-bundle/pom.xml

```diff
@@ -191,6 +216,74 @@
                   <pattern>com.fasterxml.jackson.</pattern>
                   <shadedPattern>${flink.bundle.shade.prefix}com.fasterxml.jackson.</shadedPattern>
                 </relocation>
+                <!-- The classes below in org.apache.hadoop.metrics2 package come from
```

Comment:
why

Reply:
There are classes from `hbase-hadoop2-compat` and `hbase-hadoop-compat` implementof `org.apache.hadoop.metrics2` classes in `hadoop-common`.  We cannot shade all classes in `org.apache.hadoop.metrics2` which cause NoClassDef of hadoop classes.  So have to pick and choose here for HBase specific ones.